mem_sql/pom.xml

```diff
@@ -259,12 +259,10 @@
                             <key>com.splicemachine.enableLegacyAsserts</key>
                             <value>true</value>
                         </systemProperty>
-                        <!-- Uncomment the following to dump activation class files for mem platform to the mem_sql directory.
```

Comment:
The other comment pointed to the wrong code.  dumpClassFile is the property we don't want to leave uncommented:

Reply:
Very good point, thanks for catching that!